src/frontends/tensorflow_lite/src/frontend.cpp

```diff
@@ -268,7 +268,7 @@ void FrontEnd::normalize(const std::shared_ptr<ov::Model>& function) const {
     manager.register_pass<ov::frontend::tensorflow_lite::pass::TFLQuantizeResolver>();
     manager.register_pass<ov::frontend::tensorflow_lite::pass::Rfft2dSimplifier>();
     manager.register_pass<ov::pass::TransposeSinking>();
-    manager.register_pass<ov::pass::TransposeSinkingGeneral>();
+    manager.register_pass<ov::pass::transpose_sinking::TSGeneral>();
```

Comment:
To me `ov::pass::TransposeSinking` looks better. You can include all particular TS passes into `transpose_sinking` namespace and remain the main entry on `pass` namespace.

Reply:
unfortunately, we already have ov::pass::TransposeSinking (old common transformation), see line 270
We will be able to replace it in the next Release

Reply:
I see what about `TransposeSinkingExtended` or `TransposeSinkingAdvanced` names?

Reply:
I believe we can leave both options:
ov::pass::TransposeSinkingGeneral (the alias)
ov::pass::transpose_sinking::TSGeneral

Reply:
it will be fixed in the next PR